vtex-sites/base.store
# fix: properly assign product/sku ids and reference id to 'add_to_cart' analytics event

## What's the purpose of this pull request?
This PR fixes the values of properties that were being sent to the analytics when the 'add_to_cart' event was triggered.

## How it works? 
It changes the values of the following fields:
- `item_id`: This property receives the value of the product group id, that is, the id of the "generic item" - e.g. `"SNEAKER"` (id `1`)
- `item_variant`: This property receives the value of the product variant - e.g. a variant of a `"SNEAKER"`: `WHITE SIZE 43`(id `234`)
- `product_reference_id`: This property receives the value of the `gtin` field for the product (Global Trade Item Number).

## How to test it?
`yarn develop`, then add a product to cart. On the console, check the value of `window.dataLayer`. Verify that the fields are correctly assigned and referent to the product that was added to cart.

Sample expected payload:
```json
{
  currency: "USD"
  item_brand: "Lacoste"
  item_id: "14"
  item_name: "Zip Detail Nylon and Cotton Twill Bucket Hat"
  item_variant: "123"
  price: 135
  product_reference_id: "80380901XS"
  quantity: 1
  sku_name: "Zip Detail Nylon and Cotton Twill Bucket Hat - XS"
}
```

## References
@tlgimenes
